.github/workflows/build.yml

```
@@ -0,0 +1,24 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: 8
+          distribution: 'adopt'
```

```suggestion
```suggestion
        with:
          distribution: 'zulu'
          java-version: '11'
          java-package: jdk
```
```

---

Thanks. Done.